includes/definitions/discovery/socomec-ups.yaml

```diff
@@ -0,0 +1,275 @@
+mib: SOCOMECUPS-MIB
+modules:
+    os:
+        hardware: SOCOMECUPS-MIB::upsIdentModel.0
+        version: SOCOMECUPS-MIB::upsIdentAgentSoftwareVersion.0
+        version_regex: '/v(?<version>[0-9]+.[0-9]+)/'
+        serial: SOCOMECUPS-MIB::upsIdentSerialNumber.0
+    sensors:
+        pre-cache:
```

Comment:
Don't use Pre-Cache here, it makes the code slower.

Reply:
Done